files/class-vip-filesystem.php

```diff
@@ -221,6 +221,7 @@ protected function validate_file_type( $file_path ) {
 		if ( is_wp_error( $result ) ) {
 			if ( 'invalid-file-type' !== $result->get_error_code() ) {
 				trigger_error(
+					// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
Not sure if we should escape here, or leave ignored? If escape, which wp escape function would be a good fit here?

Reply:
It should be fine without it. Even single quotes passed in via get_error_message would be handled appropriately by sprintf.